python-sdk/tests/airflow_test/test_datasets.py

```diff
@@ -1,7 +1,7 @@
 import os
 from unittest import mock
 
-import airflow
+import airflow_test
```

Comment:
@kaxil i've been running into issues in pycharm where it can't run tests in the IDE due to conflicts between the tests and the imported packages. I wonder if OSS users might run into the same issue.

Reply:
@pankajastro Also faced this issue recently.